src/main/java/com/example/onlinebookstore/config/SecurityConfig.java

```diff
@@ -36,7 +38,8 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
         return http.cors(AbstractHttpConfigurer::disable)
                 .csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests(
-                        auth -> auth.requestMatchers("/auth/**")
+                        auth -> auth.requestMatchers(AUTHORIZATION_URL,
+                                        "/swagger-ui/**", "/v3/api-docs/**", ERROR_URL)
```

Comment:
If you created constants for AUTHORIZATION_URL please make constants for other URLs to be consistent